src/Integration/Laravel/Queue/src/AsyncAwsSqsQueue.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace AsyncAws\Illuminate\Queue;
+
+use AsyncAws\Illuminate\Queue\Job\AsyncAwsSqsJob;
+use AsyncAws\Sqs\Enum\QueueAttributeName;
+use AsyncAws\Sqs\SqsClient;
+use Illuminate\Contracts\Queue\Queue as QueueContract;
+use Illuminate\Queue\Queue;
+use Illuminate\Support\Str;
+
+class AsyncAwsSqsQueue extends Queue implements QueueContract
+{
+    /**
+     * The Amazon SQS instance.
+     *
+     * @var \AsyncAws\Sqs\SqsClient
```

Comment:
`@var SqsClient`

If we copy/paste the code from somewhere else, we should add a comment in this class.
Otherwise, we should use our own "coding style" and remove those useless comments

Reply:
Thank you